agenda-services/src/main/java/org/exoplatform/agenda/util/NotificationUtils.java

```diff
@@ -241,4 +270,32 @@ private static final void setLasModifiedTime(NotificationInfo notification, Temp
                                                                      TimeConvertUtils.YEAR));
   }
 
+  private static final String getEventCreatorOrModifierUserName(long idUser) {
+    IdentityManager identityManager = ExoContainerContext.getService(IdentityManager.class);
+    Identity identity = identityManager.getIdentity(String.valueOf(idUser));
+    return identity.getProfile().getFullName();
+  }
+
+  private static String getUserAbsoluteURI(String id) {
+    String currentSite = CommonsUtils.getCurrentSite().getName();
+    String currentDomain = CommonsUtils.getCurrentDomain();
+    if (!currentDomain.endsWith("/")) {
+      currentDomain += "/";
+    }
+    return currentDomain + "portal/" + currentSite + "/profile/" + id;
+  }
+
+  private static ZoneId getZoneId(String idUser) {
```

Comment:
Add this generic method in Utils class and rename it to `getUserZoneId`

Reply:
ok